app/views/articles/manage.html.erb

```diff
@@ -51,23 +51,23 @@
       <%= f.hidden_field :article_id, value: @article.id %>
       <%= f.hidden_field :group, value: "experience_level" %>
       <span class="button-section button-section-newbie">
-        <button value="1" name="rating_vote[rating]" class="level-rating-button <%= "selected" if @rating_vote&.rating == 1.0 %>">1</button>
-        <button value="2" name="rating_vote[rating]" class="level-rating-button <%= "selected" if @rating_vote&.rating == 2.0 %>">2</button>
-        <button value="3" name="rating_vote[rating]" class="level-rating-button <%= "selected" if @rating_vote&.rating == 3.0 %>">3</button>
+        <button value="1" name="rating_vote[rating]" class="level-rating-button <%= "selected" if @rating_vote&.rating.to_d == 1.0.to_d %>">1</button>
```

Comment:
I was getting a rubocop issue on these. I'm happy to revert this particular change if y'all would prefer that.